docs/sphinx/using/cudaq/platform.rst

```diff
@@ -104,32 +116,134 @@ should be used.
 Last but not least, the compiled executable (C++) or Python script needs to be launched with an appropriate MPI command, 
 e.g., :code:`mpirun`, :code:`mpiexec`, :code:`srun`, etc.
+
+Multi-GPU QPU simulation
++++++++++++++++++++++++++++
+
+As shown in the above examples, the :code:`nvidia-mqpu` platform enables
+multi-QPU distribution whereby each QPU is simulated by a single NVIDIA GPU.
+
+In use cases where the number of qubits requires
+the multi-node multi-GPU simulator (:code:`nvidia-mgpu`) as the simulated QPU, 
+multi-QPU distribution capability is provided by the remote simulator target (:code:`remote-sim`).
+
+Specifically, this :code:`remote-sim` platform encapsulates simulated QPUs as independent
+HTTP REST server instances (:code:`cudaq-qpud`). 
+
+For instance, on a machine with 8 NVIDIA GPUs, one may wish to partition those GPUs into
+4 virtual QPU instances, each manages 2 GPUs.
+
+
+.. code-block:: console
+    
+    CUDA_VISIBLE_DEVICES=0,1 mpirun -np 2 cudaq-qpud --port <QPU 1 TCP/IP port number>
+    CUDA_VISIBLE_DEVICES=2,3 mpirun -np 2 cudaq-qpud --port <QPU 2 TCP/IP port number>
+    CUDA_VISIBLE_DEVICES=4,5 mpirun -np 2 cudaq-qpud --port <QPU 3 TCP/IP port number>
+    CUDA_VISIBLE_DEVICES=6,7 mpirun -np 2 cudaq-qpud --port <QPU 4 TCP/IP port number>
+
+
+In the above code snippet, four :code:`nvidia-mgpu` daemons are started in MPI context via the :code:`mpirun` launcher.
+This activates MPI runtime environment required by the :code:`nvidia-mgpu` backend. Each QPU daemon is assigned a unique 
+TCP/IP port number via the :code:`--port` command-line option. The :code:`CUDA_VISIBLE_DEVICES` environment variable restricts the GPU devices 
+that each QPU daemon sees so that it targets specific GPUs. 
+
+With these invocations, each virtual QPU is locally addressable at the URL `localhost:<port>`. 
+
+.. warning:: 
+
+    There is no authentication required to communicate with this server app. 
+    Hence, please make sure to either (1) use a non-public TCP/IP port for internal use or 
+    (2) use firewalls or other security mechanisms to manage user access. 
+
+User code can then target these QPUs for multi-QPU workloads, such as asynchronous sample or observe shown above for the :code:`nvidia-mqpu` platform.
+
+.. tab:: C++
+
+    .. code-block:: console
+
+        nvq++ distributed.cpp --target remote-sim --remote-sim-url localhost:<port1>,localhost:<port2>,localhost:<port3>,localhost:<port4> --remote-sim-backend nvidia-mgpu
+
+
+.. tab:: Python
+
+     .. code:: python 
+
+        cudaq.set_target("remote-sim", url="localhost:<port1>,localhost:<port2>,localhost:<port3>,localhost:<port4>", backend="nvidia-mgpu")
+    
+
+Each URL is treated as an independent QPU, hence the number of QPUs (:code:`num_qpus()`) is equal to the number of URLs provided. 
+The multi-node multi-GPU simulator backend (:code:`nvidia-mgpu`) is requested via the :code:`--remote-sim-backend` command-line option.
```

Comment:
Along with renaming the target to `remote-mqpu`, I would suggest to also rename the --remote-sim-backend to --remote-qpu-backend. In the future, this could be any one of the supported backends, simulated or actual QPU.

Reply:
Due to `nvq++` command-line argument handling policy, target-specific arguments must be in the form: `--<target>-<option> <value>`. Hence, it will be `--remote-mqpu-backend`.